pyproject.toml

```diff
@@ -33,10 +36,6 @@ dependencies = [
     "pytz",
     "pyyaml",
     "certifi",
-    "pyarrow",
```

Comment:
Do we no longer depend on pyarrow, polars, pandas etc?

Reply:
We do when you specify the `btrdb[data]` installation. The base installation does not necessarily need it. We have import guards on methods that might use features like pyarrow, etc.

Reply:
Oic, that makes sense. Thanks!